docs/pages/get-started/faq.md

```diff
@@ -324,33 +325,57 @@ See the [XMTP brand guidelines](https://github.com/xmtp/brand) GitHub repo.
+
+Push notifications are particularly valuable for messaging apps, as they help ensure users don't miss important communications and can respond promptly to new messages.
+
+### Does XMTP support group chat?
+
+XMTP supports group chat in React Native, Android, iOS, and Node.js SDKs.
+
+:::info[For more info]
+See [Build group chat with MLS and XMTP](/groups/build-group-chat)
+:::
+
+To learn more about when group chat will be available for the web, see the XMTP Developer Community Call about [wen group chat web](https://www.youtube.com/watch?v=qnCeXsqqKcY).
+
+### Is there a way to get a list of all current XMTP-enabled wallets?
+
+XMTP doesn't provide a direct method to fetch all XMTP-enabled wallet addresses in bulk. You might consider using a third-party service to do so instead of building the functionality yourself.
+
+[Blaze CRM](https://www.withblaze.app/) is one known service. It compiles lists of XMTP addresses by querying the XMTP `canMessage` API over time.
+
+The XMTP ecosystem is continually evolving. If you know of other reliable services providing this functionality, please consider sharing them in the [XMTP Community Forums](https://community.xmtp.org/c/development/third-party-tools/74).
 
 ## Network
 
 ### Is the XMTP network decentralized?
 
-Currently, Ephemera (the company) operates all of the network nodes in the two available XMTP network environments: `dev` and `production`.
+Currently, [Ephemera](https://ephemerahq.com/) (the company) operates all of the network nodes in the two available XMTP network environments: `dev` and `production`.
 
 These network nodes operate in US jurisdiction in compliance with Office of Foreign Assets Control (OFAC) sanctions and Committee on Foreign Investment in the United States (CFIUS) export compliance regulations. Accordingly, IP-based geoblocking is in place for the following countries/territories:
 
 - Cuba
 - Iran
-- North Korea
+- North Korea 
 - Syria
 - The Crimea, Donetsk People’s Republic, and Luhansk People’s Republic regions of Ukraine
 
 To explore the software for the nodes that currently form the XMTP network, see the [xmtp-node-go repo](https://github.com/xmtp/xmtp-node-go).
 
-Ephemera is working toward a phased decentralization of the network.
+Ephemera is in the process of building the replication and incentive mechanisms required to decentralize the network. These mechanisms include:
+
+- An on-chain smart contract node registry, which lists the currently active set of XMTP nodes
+- An off-chain broadcast network of nodes run by a set of independent third parties, which handles the majority of the messaging data (loosely ordered, low latency)
+- A series of on-chain smart contracts on an XMTP L3, which handles account information and the most critical messaging data (strictly ordered, medium latency)
```

Comment:
Jesse from Base is pretty dogmatic about "onchain" being one word. I don't have a strong opinion, but I think we should do it his way if no one else feels strongly otherwise.